### PR TITLE
Acl 2021 ingestion

### PR DESCRIPTION
**Pull request by xinru1414**

### CODE REVIEW

**Comment by nschneid** [comment]
I see 4 .yaml files updated for SIGs. SIGMORPHON should be added to sigmorphon.yaml, right? (And possibly other SIGs for other workshops...)

File: data/xml/2021.lchange.xml [hunk 1]
```diff
@@ -0,0 +1,107 @@
+<?xml version='1.0' encoding='UTF-8'?>
+<collection id="2021.lchange">
+  <volume id="1" ingest-date="2021-07-20">
+    <meta>
+      <booktitle>Proceedings of The 2nd International Workshop on Computational Approaches to Historical Language Change 2021</booktitle>
+      <editor><first>workshop chair</first><last>Nina Tahmasebi</last></editor>
```
Comment (danielgildea) [inline data/xml/2021.lchange.xml:6]: "workshop chair" in name field

File: data/xml/2021.bppf.xml [hunk 2]
```diff
@@ -0,0 +1,60 @@
+<?xml version='1.0' encoding='UTF-8'?>
+<collection id="2021.bppf">
+  <volume id="1" ingest-date="2021-07-20">
```
Comment (mjpost) [inline data/xml/2021.bppf.xml:3]: Ingestion date should be 2021-07-25 on all volumes.

Comment (mjpost) [reply data/xml/2021.bppf.xml]: I just fixed this:

```bash
$ cd data/xml
$ perl -pi -e "s/2021-07-20/2021-07-25/" *.xml
```

**Comment by mjpost** [comment]
I thought I wrote a comment here but can't find it. The following are TODOs:

- [ ] Remove `2021.tutorials.xml`, reingest with venue `acl` and volume `tutorials`
- [ ] Add all workshops to `joint.xml`
- [ ] Reorder volumes as long papers, short papers, demos, tutorials, SRW
- [ ] Adjust all addresses to "Online" only

File: data/xml/2021.lchange.xml (hunk 1 above)
Comment (xinru1414) [reply data/xml/2021.lchange.xml:6]: @danielgildea thank you for pointing this out!

**Comment by xinru1414** [comment]
> I see 4 .yaml files updated for SIGs. SIGMORPHON should be added to sigmorphon.yaml, right? (And possibly other SIGs for other workshops...)

**Comment by xinru1414** [comment]
> I see 4 .yaml files updated for SIGs. SIGMORPHON should be added to sigmorphon.yaml, right? (And possibly other SIGs for other workshops...)

@nschneid Yes that's right. Thank you for catching this!

**Comment by mjpost** [comment]
I fixed tutorials and the volume ordering

**Review by mjpost — APPROVED** [review]

**Pull request merged**